.github/workflows/ci.yml

```diff
@@ -48,7 +48,7 @@ jobs:
           working-directory: packages/cli/
 
           # Optional: golangci-lint command line arguments.
-          # args: --issues-exit-code=0
+          args: --timeout=5m0s
```

Comment:
How is this change related to workflow logs?

Reply:
For whatever reason, the linter is sometimes taking more than a minute to pass. I don't think it's related to WF logs, but if it's not increased it will lead to infrequent build failures.